packages/js-sdk/src/getConfig.js

```diff
@@ -9,17 +9,16 @@ const getConfigData = (chainId) => {
     if (!networkData) {
         console.log(`no metadata found for network with chainId ${chainId}`);
         return {
-            nativeTokenSymbol: "",
-            resolverAddress: "",
+            isTestnet: true,
```

Comment:
Observation: this introduces changes to js-sdk which will be unpublished.

It's time to piggyback this change:

- Remove ""ops-scripts/**/*"," from package.json
- dev-scripts remain published, since it should be for developers.
- to use ops-scripts, one should always check out the git repo branch.

Reply:
I changed it.